components/dialog/metadata/dialog.yml

```diff
@@ -14,13 +14,22 @@ examples:
     markup: |
       <button class="spectrum-Button spectrum-Button--sizeM spectrum-Button--outline spectrum-Button--staticWhite spectrum-CSSExample-overlayShowButton" onclick="openDialog(this.nextElementSibling)"><span class="spectrum-Button-label">Open Small Dialog</span></button>
 
-      <div class="spectrum-Modal-wrapper spectrum-CSSExample-dialog">
-        <div class="spectrum-Modal is-open">
+      <div class="spectrum-Modal-wrapper spectrum-CSSExample-modal">
+        <div class="spectrum-Modal">
```

Comment:
Curious, why remove is-open?

Reply:
Having `is-open` on page load wasn't adding the underlay styles or preventing a user from clicking on things "below" or "under" the dialog's z-index. Removing it is more steps for the user to see that dialog, but keeps the full-fledged styles & behaviors in place when the dialog is triggered (and that `is-open` gets added).

https://github.com/adobe/spectrum-css/assets/69602589/e33ed9fe-2d04-46dc-b301-245b537ccd64

I could probably refactor the `enhancements.js` file for the dialog, but thought this was a quicker fix, and didn't want to spend too much time in there if we're moving away from the docs site.